test/jdk/sun/security/pkcs11/tls/tls12/FipsModeTLS12.java

```diff
@@ -88,12 +89,11 @@ public static void main(String[] args) throws Exception {
         try {
             initialize();
         } catch (Exception e) {
-            System.out.println("Test skipped: failure during" +
-                    " initialization");
             if (enableDebug) {
                 System.out.println(e);
             }
-            return;
+            throw new SkippedException("Test skipped: failure during " +
```

Comment:
I think we should update test to fail if call to `initialize` fails with any other exception than `SkippedException` which is thrown from call to `initSecmod` if NSS library fails. So you can remove try-catch here.